src/Tasks/Microsoft.NET.Build.Tasks/build/GenerateDeps/GenerateDeps.proj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="BuildDepsJson">
+  <!--
+***********************************************************************************************
+GenerateDeps.proj
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+  
+  <!--
+    This project is built by the .NET CLI in order to create .deps.json files for .NET CLI tools.
+    Properties to be passed in by the .NET CLI:
+      - ProjectAssetsFile: Full path to the project.assets.json file for the tool under the NuGet .tools folder
+      - ToolName: The simple name of the tool DLL, for example, "dotnet-mytool"
+      - AdditionalImport: The full path to the .props file from the platform package which will be imported, which
+        should include the PackageConflictPlatformManifests file.      
+        This is a workaround until NuGet can generate .props and .targets files for imports from packages referenced
+        by tools, which is tracked by https://github.com/NuGet/Home/issues/5037.
+  -->
+
+  <Import Project="$(AdditionalImport)"
+          Condition=" '$(AdditionalImport)' != '' And Exists($(AdditionalImport))" />
+  
+  <PropertyGroup>
+    <TargetFramework>netcoreapp$(BundledNETCoreAppTargetFrameworkVersion)</TargetFramework>
```

Comment:
This this always going to hold true?  What if the project.assets.json file was restored with `netcoreapp1.0`?  Won't `GenerateBuildDependencyFile` raise an error because there isn't a `netcoreapp2.0` target in the lock file?